### ISSUE TITLE
Get in contact

### ISSUE THREAD

**Issue by rikyeahh** [ekohilas/kingdomino#1]
I'm trying to get in contact with you but I can't find your email. I'm trying to use this library for an AI project, but I've (probably) found a bug. I'm totally willing to help, want to get in touch and cooperate?

**Comment by ekohilas** [comment]
Of course! Yes unfortunately I don’t have a public email. Are you able to message me on LinkedIn or Facebook?

**Comment by rikyeahh** [comment]
Yeah I've added you on linkedin. Thanks!
